rpc/internal/perf_service.go

```diff
@@ -106,6 +106,19 @@ func (srv *perfService) AttachRollups(ctx context.Context, rollupData *RollupDat
 		return nil, newRPCError(codes.InvalidArgument, errors.Wrapf(err, "problem attaching rollup data for perf result '%s'", record.ID))
 	}
 
+	id := model.PerformanceResultSeriesId{
+		Project: record.Info.Project,
+		Variant: record.Info.Variant,
+		Task:    record.Info.TaskName,
+		Test:    record.Info.TestName,
+	}
+
+	processingJob := units.NewRecalculateChangePointsJob(id)
+	err := srv.env.GetRemoteQueue().Put(ctx, processingJob)
```

Comment:
combine 116 and 117

Reply:
Go fmt file is pushing them back apart for me (line length?). Not for you?